README.md

```diff
@@ -1 +1,43 @@
-# quantified_self
+#### Quantified Self
+
+## Team Members: Jonathan Kidd & Lauren Oliveri
+
+### API Endpoints
+
+## Food CRUD
+
+# List of All Foods
+
+`GET /api/v1/foods`
+
+# Show Particular Food
+
+`GET /api/v1/foods/:id`
+
+# Create a New Food
+
+`POST /api/v1/foods?name=[]&calories=[]`
+
+# Update an Existing Food
+
+`PUT /api/v1/foods/:id?name=[]&calories=[]`
+
+# Delete a Food
+
+`DELETE /api/v1/foods/:id`
+
+## Updating Diary
+
+# List All Meals
+
+`GET /api/v1/diary`
+
+# Add Food to Meal
+
+`POST /api/v1/diary?food_id=[]&meal_id=[]`
+
+# Remove Food from Meal
+
+`DELETE /api/v1/diary?food_meal_id=id`
```

Comment:
Thanks for taking a look @case-eee ! I updated it, you're correct, we just had a bit of an inconsistency there.

@jk1dd - do you want to merge this when you get a chance?